package.json

```diff
@@ -3,7 +3,6 @@
   "version": "7.3.0",
   "description": "A React component to execute a function whenever you scroll to an element.",
   "main": "build/index.js",
-  "module": "build/index.mjs",
```

Comment:
I believe the rollup config has a direct dependency on this field, so you will need to modify that as well.

Reply:
Ah, thanks. I had missed that.